lib/view-models/vim-command-mode-input-view.coffee

```diff
@@ -4,16 +4,21 @@ module.exports =
 class VimCommandModeInputView extends View
   @content: ->
     @div class: 'command-mode-input', =>
-      @div class: 'editor-container', outlet: 'editorContainer', =>
-        @subview 'editor', new TextEditorView(mini: true)
+      @div class: 'editor-container', outlet: 'editorContainer'
 
   initialize: (@viewModel, opts = {})->
     if opts.class?
       @editorContainer.addClass opts.class
 
     if opts.hidden
+      # TODO: this no longer works with atom-text-editor
       @editorContainer.addClass 'hidden-input'
```

Comment:
The `hidden-input` class was a special case in the deprecated TextEditorView. I'm not sure what the equivalent is for TextEditorElement